cocos/3d/CCMesh.cpp

```diff
@@ -350,6 +350,7 @@ void Mesh::draw(Renderer* renderer, float globalZOrder, const Mat4& transform, u
 
 
     _meshCommand.setSkipBatching(isTransparent);
+    _meshCommand.setTransparent(isTransparent);
```

Comment:
The default transparent property of isTransparent in RenderCommand is true. So it treats all Sprite3D as transparent. It does not work when we can not sort them correctly.

I should correct it here even if I send another PR to fix it. Because we must get correct rendering result here. I will sender another PR if you like.